### ISSUE TITLE
TypeScript issue

### ISSUE THREAD

**Issue by lucienimmink** [Kamiapp-fr/vite-plugin-cem#1]
Latest vite; using the starter project lit-ts

```
Type 'Plugin' is not assignable to type 'PluginOption'.
  Type 'Plugin' is not assignable to type 'Plugin_2'.
    Types of property 'apply' are incompatible.
      Type '"build" | "serve" | ((config: import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").UserConfig, env: import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").ConfigEnv) => boolean) | undefined' is not assignable to type '"build" | "serve" | ((config: import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").UserConfig, env: import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").ConfigEnv) => boolean) | undefined'.
        Type '(config: UserConfig, env: ConfigEnv) => boolean' is not assignable to type '"build" | "serve" | ((config: UserConfig, env: ConfigEnv) => boolean) | undefined'.
          Type '(config: import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").UserConfig, env: import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").ConfigEnv) => boolean' is not assignable to type '(config: import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").UserConfig, env: import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").ConfigEnv) => boolean'.
            Types of parameters 'config' and 'config' are incompatible.
              Type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").UserConfig' is not assignable to type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").UserConfig'.
                Types of property 'plugins' are incompatible.
                  Type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").PluginOption[] | undefined' is not assignable to type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").PluginOption[] | undefined'.
                    Type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").PluginOption[]' is not assignable to type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").PluginOption[]'.
                      Type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").PluginOption' is not assignable to type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").PluginOption'.
                        Type 'Plugin_2' is not assignable to type 'PluginOption'.
                          Type 'Plugin_2' is not assignable to type 'Plugin'.
                            Types of property 'apply' are incompatible.
                              Type '"build" | "serve" | ((config: import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").UserConfig, env: import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").ConfigEnv) => boolean) | undefined' is not assignable to type '"build" | "serve" | ((config: import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").UserConfig, env: import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").ConfigEnv) => boolean) | undefined'.
                                Type '(config: UserConfig, env: ConfigEnv) => boolean' is not assignable to type '"build" | "serve" | ((config: UserConfig, env: ConfigEnv) => boolean) | undefined'.
                                  Type '(config: import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").UserConfig, env: import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").ConfigEnv) => boolean' is not assignable to type '(config: import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").UserConfig, env: import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").ConfigEnv) => boolean'.
                                    Types of parameters 'config' and 'config' are incompatible.
                                      Type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").UserConfig' is not assignable to type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").UserConfig'.
                                        Types of property 'plugins' are incompatible.
                                          Type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").PluginOption[] | undefined' is not assignable to type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").PluginOption[] | undefined'.
                                            Type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").PluginOption[]' is not assignable to type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").PluginOption[]'.
                                              Type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite-plugin-cem/node_modules/vite/dist/node/index").PluginOption' is not assignable to type 'import("D:/Workspace/commerce-components-frontend/node_modules/vite/dist/node/index").PluginOption'.
                                                Type 'PluginOption[]' is not assignable to type 'PluginOption'.ts(2322)
```

**Comment by EmilienLeroy** [comment]
Thanks for reporting this !
I will take a look at this to fix it.

**Comment by EmilienLeroy** [comment]
Hello @lucienimmink 
This is fixed in [v0.3.0](https://github.com/Kamiapp-fr/vite-plugin-cem/releases/tag/v0.3.0) ! 😄
